docs/operator-manual/high_availability.md

```diff
@@ -24,50 +24,50 @@ The `--parallelismlimit` flag controls how many manifests generations are runnin
 
 ### argocd-application-controller
 
 **settings:**
 
-The `argocd-application-controller` uses `argocd-repo-server` to get generated manifests and Kubernetes API server to get actual cluster state.
+The `argocd-application-controller` uses `argocd-repo-server` to get generated manifests and Kubernetes API server to get the actual cluster state.
 
-* each controller replica uses two separate queues to process application reconciliation (milliseconds) and app syncing (seconds). Number of queue processors for each queue is controlled by
-`--status-processors` (20 by default) and `--operation-processors` (10 by default) flags. Increase number of processors if your Argo CD instance manages too many applications.
+* each controller replica uses two separate queues to process application reconciliation (milliseconds) and app syncing (seconds). The number of queue processors for each queue is controlled by
+`--status-processors` (20 by default) and `--operation-processors` (10 by default) flags. Increase the number of processors if your Argo CD instance manages too many applications.
 For 1000 application we use 50 for `--status-processors` and 25 for `--operation-processors`
 
-* The manifest generation typically takes the most time during reconciliation. The duration of manifest generation is limited to make sure controller refresh queue does not overflow.
-The app reconciliation fails with `Context deadline exceeded` error if manifest generating taking too much time. As workaround increase value of `--repo-server-timeout-seconds` and
-consider scaling up `argocd-repo-server` deployment.
+* The manifest generation typically takes the most time during reconciliation. The duration of manifest generation is limited to make sure the controller refresh queue does not overflow.
+The app reconciliation fails with `Context deadline exceeded` error if the manifest generation is taking too much time. As a workaround increase the value or `--repo-server-timeout-seconds` and
+consider scaling up the `argocd-repo-server` deployment.
 
-* The controller uses `kubectl` fork/exec to push changes into the cluster and to convert resource from preferred version into user specified version
-(e.g. Deployment `apps/v1` into `extensions/v1beta1`). Same as config management tool `kubectl` fork/exec might cause pod OOM kill. Use `--kubectl-parallelism-limit` flag to limit
+* The controller uses `kubectl` fork/exec to push changes into the cluster and to convert resources from the preferred version into the user specified version
+(e.g. Deployment `apps/v1` into `extensions/v1beta1`). The same applies for the config management tool `kubectl` fork/exec, it might cause a pod OOM kill. Use `--kubectl-parallelism-limit` flag to limit the
 number of allowed concurrent kubectl fork/execs.
```

Comment:
I think this statement is outdated and not true anymore at all, as we don't fork/exec `kubectl` anymore. Maybe we should delete this statement completely.

Reply:
Removed. :+1: